package.json

```diff
@@ -21,12 +21,12 @@
     "node": ">=10"
   },
   "dependencies": {
+    "ssb-ref": "^2.14.3",
```

Comment:
This is not great, ssb-ref shouldn't be a dependency in ssb-uri2 (just think about how it would become a dep mess to put ssb-uri2 checks inside ssb-ref) because they are both low-level libraries that do separate things. I have a suggestion instead:

Reply:
moved it back to dev deps again